x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/post_case.ts

```diff
@@ -482,14 +482,14 @@ export default ({ getService }: FtrProviderContext): void => {
           );
         });
 
-        it('400s when trying to create case with a required custom field', async () => {
+        it('400s when trying to create case without a required custom field', async () => {
```

Comment:
Ah, I see what it meant.

> Otherwise, if you do not have the required custom fields it should not throw 400s.

If you do not have required custom fields **in the payload** it should :D 

How about `'400s when creating a case with a missing required custom field'`?

Reply:
>If you do not have required custom fields in the payload it should :D

:) In the payload yes. What I meant is if you do not have required custom fields **configured**.

> How about '400s when creating a case with a missing required custom field'?
LGTM!